Makefile

```diff
@@ -0,0 +1,18 @@
+.PHONY: build test update_golden help
+.DEFAULT_GOAL := help
+
+build: ## Builds the Sonobuoy binary for your local OS
+	@echo "Building the sonobuoy binary..."
+	source ./scripts/build_funcs.sh; native
+	@echo "Complete. Use sonobuoy in the pwd (e.g. ./sonobuoy)"
```

Comment:
nit: Perhaps a statement that makes more sense would be "Use sonobuoy in the current directory" (pwd stands for print working directory)

Reply:
ha, I've always used it interchangeably with the current working directory (cwd) and intuitively assumed it was the 'present working directory'. TIL. 👍

Reply:
Looking at this build target I am going to actually swap it out for a simple `go build`. The reasoning is that I realized that using this build_func, just like the others, will reset the sonobuoy versions as if we are cutting a release.

The result will be what you experienced the other day: `sonobuoy run` will lead to setting the image to a non-existant image and you'll get ErrImagePull.